api/utils.js

```diff
@@ -0,0 +1,19 @@
+const checkAccessToken = (req, res, next) => {
+    const authHeader = req.headers.authorization;
+
+    if (authHeader) {
+        const token = authHeader.split(' ')[1];
+        if(token){
+            next();
+        } else{
+            res.sendStatus(401);
+        }
```

Comment:
T'as oublié les espaces encore 
```suggestion
        if(token) {
            next();
        } else {
            res.sendStatus(401);
        }
```